### DISCUSSION TITLE
Can someone explain the different secret options?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#3361] [Feature Question]
### Question

Can someone help explain the different options for secrets in the REST API [here](https://vercel.com/docs/rest-api/endpoints#create-one-or-more-environment-variables)?

It seems like there are:

system
secret
encrypted
plain
sensitive

Thanks 🙏

![CleanShot 2023-07-24 at 19 27 45@2x](https://github.com/vercel/community/assets/9967422/5954c9f6-c74a-49a9-800f-2b6c3aaf8baf)

**Comment by amyegan** [comment]
Hey @alexbjorlig. The best type for nearly every case, and the default used when creating a new env var via the UI, is `encrypted`. With this type, all values will be encrypted at rest. 

You could use `plain` if you wanted to store the value in plain text. The other types support legacy scenarios and experimental features, so they're not likely to be what you need.
